src/netcdf_to_zarr.py

```diff
@@ -373,12 +345,15 @@ def main():
     known_args, pipeline_args = parse_arguments(
         "Create a Zarr dataset from NetCDF files."
     )
+    if known_args.init_date != "1900-01-01" and (not known_args.from_init_date):
+        raise RuntimeError("--init_date can only be used along with --from_init_date flag.")
+
     pipeline_args.extend(['--save_main_session', 'True'])
 
-    if fs.exists(known_args.output_path):
-        raise ValueError(f"{known_args.output_path} already exists")
+    # if fs.exists(known_args.output_path):
```

Comment:
If this is not required then we should remove this instead of commenting ?

Reply:
We need this. Will uncomment.